detector/other/SingleShape_geo.cpp

```diff
@@ -0,0 +1,63 @@
+//==========================================================================
+//  AIDA Detector description implementation 
+//--------------------------------------------------------------------------
+// Copyright (C) Organisation europeenne pour la Recherche nucleaire (CERN)
+// All rights reserved.
+//
+// For the licensing terms see $DD4hepINSTALL/LICENSE.
+// For the list of contributors see $DD4hepINSTALL/doc/CREDITS.
+//
+// Author     : M.Frank
+//
+//==========================================================================
+
+// Edited by A. Ilg, to have only the CAD import left and nothing else
```

Comment:
Can you add one more comment line here explaining what this code does?

Reply:
Removed file